src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -138,6 +138,10 @@ func runServerProxy(cfg *config.Config) {
 			ServerCertFile: config.InConfigDir("servercert.pem"),
 		},
 		AllowedPorts: []int{80, 443, 8080, 8443, 5222, 5223, 5228},
+
+		// We allow all censored countries plus us, es and mx because we do work
```

Comment:
Super-nit: Countries are normalized to uppercase so I'd spell them that way in this comment to reinforce the point.